randovania/games/am2r/json_data/Golden Temple.txt

```diff
@@ -704,6 +706,9 @@ Extra - minimap_data: [{'x': 62, 'y': 12}, {'x': 63, 'y': 12}, {'x': 64, 'y': 12
           Mid-Air Morph (Beginner) or Tunnel Climb
   > Pickup (Energy Tank)
       After Area 1 - Golden Temple 1 Orb Puzzle
+  > Event - 1Orb
```

Comment:
assuming you hit the event with wave, how do you "get out"? from what i can see, itd still logically expect boombs for you i the current state

Reply:
It seems the I missed the return path when working on this, intended to have the same requirements as activating with wave (`wave & knowledge`) to get out. Will correct.